app.js

```diff
@@ -0,0 +1,104 @@
+function allBooks() {
```

Comment:
- [ ] Your application is throwing an error and therefore a CRUD operation cannot be performed on your application. Kindly see the screenshot below and debug your code accordingly. The next code reviewer will review make a **full review** again when this error is corrected.

**Error in code:**
![image](https://user-images.githubusercontent.com/12745474/133386506-d6452369-b578-4573-9dae-85363c7f7220.png)